added links to examples for most createLayer options

csobier opened the pull request:
For [Docs issue#685](https://github.com/CartoDB/docs/issues/685).  The people want more examples!

@alonsogarciapablo , I searched through the entire CartoDB.js repo for any code related examples specific for these createLayer options.  I think I found most of them, can you review?

Also, I would still like to find complete code examples for the following options: `https`, `tooltip`, `filter`. Please advise if you know of any, or can help me create them.

alonsogarciapablo commented on doc/api_methods.md:
```diff
@@ -142,15 +142,15 @@ layerSource | contains information about the layer. It can be specified in multi
 &#124;_ https | force https
-&#124;_ refreshTime | if is set, the layer is refreshed each refreshTime milliseconds.
-&#124;_ infowindow | set to false if you want to disable the infowindow (enabled by default).
+&#124;_ refreshTime | if is set, the layer is refreshed each refreshTime milliseconds.<br/><br/>For a code example of refreshTime, see ["layer_definition.spec.js"](https://github.com/CartoDB/cartodb.js/blob/60285edb65f6575c8a34799fde55308751251c5f/test/spec/geo/layer_definition.spec.js).
```
I don't think this is a good reference :(. If examples are going to be part of the documentation I think you should prepare some stable and reliable examples that we can point to (or include the code snippet as part of the docs).

csobier replied:
Okay, I need developer input for these examples :-)

csobier commented:
Okay @alonsogarciapablo , I updated some of the existing CartoDB.js examples to the latest version and added some new ones (based on some old bl.ocks). So now each createLayer description provides a link to an example.

Note: The only one I question is the "filters" option- I'm not sure why you would use this? Regardless, are you okay with these links and examples now? Let me know if I need to edit or fix anything.

csobier commented:
Included examples created in PR #1112 .

csobier commented:
@alonsogarciapablo , do you have any clarification about why one would use the "filters" options? If not, are you signing-off on these enhancements?

alonsogarciapablo commented:
:+1:
